static/src/javascripts/projects/commercial/modules/hosted/video.js

```diff
@@ -111,6 +128,50 @@ define([
                                 }
                             });
                         });
+
+                        if ($hostedNext.length) {
+                            //on desktop show the next video link 10 second before the end of the currently watching video
+                            if (contains(['desktop', 'leftCol', 'wide'], detect.getBreakpoint())) {
+
+                                var $timer = $('.js-autoplay-timer');
+                                var nextVideoPage;
+
+                                if ($timer.length) {
+                                    nextVideoPage = $timer.data('next-page');
+
+                                    bean.on(document, 'click', $('.js-autoplay-cancel'), function () {
+                                        cancelAutoplay($hostedNext);
+                                    });
+                                    player.one('timeupdate', function () {
+
```

Comment:
Remove this empty line and move it before `player.one('timeupdate', function () {`